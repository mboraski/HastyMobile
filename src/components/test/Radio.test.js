// TODO: Remove when tests implemented
test.only('real recognize real', () => {
    expect('real').toBe('real');
});
// import 'react-native';
// import React from 'react';
// import { shallow } from 'enzyme';
//
// import Radio from '../Radio';
//
// describe('Radio', () => {
//     it('renders correctly true', () => {
//         const render = shallow(<Radio input={{ value: true }} />);
//         expect(render).toMatchSnapshot();
//     });
//     it('renders correctly false', () => {
//         const render = shallow(<Radio input={{ value: false }} />);
//         expect(render).toMatchSnapshot();
//     });
// });
