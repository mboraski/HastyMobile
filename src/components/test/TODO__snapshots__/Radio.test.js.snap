// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Radio renders correctly false 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  onPress={[Function]}
  value={false}
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "paddingHorizontal": 15,
        "paddingVertical": 30,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "white",
            "borderColor": "#000",
            "borderRadius": 11.5,
            "borderWidth": 0.5,
            "height": 23,
            "justifyContent": "center",
            "marginRight": 15,
            "overflow": "hidden",
            "width": 23,
          },
          false,
        ]
      }
    >
      <Icon
        allowFontScaling={false}
        color="#fff"
        name="check"
        size={18}
        style={
          Array [
            undefined,
            false,
          ]
        }
      />
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "fontSize": 16,
        }
      }
    />
  </View>
</TouchableOpacity>
`;

exports[`Radio renders correctly true 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  onPress={[Function]}
  value={true}
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "paddingHorizontal": 15,
        "paddingVertical": 30,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "white",
            "borderColor": "#000",
            "borderRadius": 11.5,
            "borderWidth": 0.5,
            "height": 23,
            "justifyContent": "center",
            "marginRight": 15,
            "overflow": "hidden",
            "width": 23,
          },
          Object {
            "backgroundColor": "black",
          },
        ]
      }
    >
      <Icon
        allowFontScaling={false}
        color="#fff"
        name="check"
        size={18}
        style={
          Array [
            undefined,
            undefined,
          ]
        }
      />
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "fontSize": 16,
        }
      }
    />
  </View>
</TouchableOpacity>
`;
